torch/_inductor/codecache.py

```diff
@@ -825,7 +825,15 @@ def __init__(
                     from triton.runtime.autotuner import Autotuner
 
                     kernel = kernel_side_table.get_kernel(node.kwargs["kernel_idx"])
+                    configs = None
                     if isinstance(kernel, Autotuner):
+                        if kernel.configs:
+                            configs = str(
+                                sorted(
+                                    sorted(c.all_kwargs().items())
```

Comment:
dropping the keys is not safe normally since a : 1 and b : 1 should be different but in this maybe fine?

Reply:
We're not dropping the keys? `.items()` generates key/value tuples.